src/scf_module.F90

```diff
@@ -858,11 +860,29 @@ subroutine scf(env, mol, wfn, basis, pcem, xtbData, solvation, &
       end if
 
    endif printing
-
-   ! ------------------------------------------------------------------------
-   ! get Wiberg bond orders
-   call get_wiberg(mol%n, basis%nao, mol%at, mol%xyz, wfn%P, S, wfn%wbo, &
-      & basis%fila2)
+   
+   !--------------------!
+   ! Wiberg bond orders !
+   !--------------------!
+   
+   ! restricted case !
+   if (wfn%nopen == 0) then
+      
+      call get_wiberg(mol%n, basis%nao, mol%at, mol%xyz,wfn%P, S, wfn%wbo,basis%fila2)
+   
+   ! unrestricted case !
```

Comment:
That would be (restricted) open-shell rather than unrestricted. For unrestricted two spin channels are necessary